app/views/assessments/index.html.erb

```diff
@@ -66,7 +66,7 @@
           <% end %>
 
           <% if @cud.instructor? %>
-           <%= link_to "<div class='row metrics'><i class='material-icons left'>assessment</i> Risk Metrics</div>".html_safe, course_metrics_path, title: "View student metrics", :class => "btn btn-large red darken-3" %>
+           <%= link_to "<div class='row metrics'><i class='material-icons left'>assessment</i> Student Metrics</div>".html_safe, course_metrics_path, title: "View Metrics for Identifying Students in Need", :class => "btn btn-large red darken-3" %>
```

Comment:
The wording sounds a bit off. Maybe change it to just "Metrics for Identifying Students in Need"